File: Python Projects/mario.py
from cs50 import get_int

#Gets input from user on loop until correct input is given
while True:
    h = get_int("Height: ")
    if h >= 1 and h <= 8:
        break;
        
#Creates half-pyramids
i = 0
for i in range(h):
    
#Spaces
    j=0
    for j in range(h-i):
        print(" ", end = "")
        j+=1
    
#Hashes(L)
    d = 0
    while True:
        print("#", end = "")
        d+=1
        if d > i:
            break
        
#Space in between
    print("  ", end = "")
    
#Hashes(R)
    d = 0
    while True:
        print("#", end = "")
        d+=1
        if d > i:
            break
    i+=1
    print("")

File: C Projects/mario.c
#include <cs50.h>
#include <stdio.h>


//main function
int main(void)
{
    
    int h = 0;
    
    
     //ask for input within parameters
     do
     {
     h = get_int("Height: ");
     }
     while (h < 1 || h > 8);
    
     for (int i = 0; i < h; i++)
     {
      
     //spaces
     
     for(int b = 1; b < h-i; b++ )
     {
           printf(" ");
      }
       
     //hashes (L)
     for(int d = 0; d <= i; d++)
      {
           printf("#");
      }
       
      //space between hashes
      printf("  ");
       
      //hashes (R)
      for(int d = 0; d <= i; d++)
      {
           printf("#");
      }
     
      //next line
        printf("\n");
      }
}